openhtf/io/frontend/__main__.py

```diff
@@ -225,40 +214,65 @@ def post(self, host, port, prompt_id):
     msg = json.JSONEncoder().encode(
         {'id': prompt_id, 'response': self.request.body})
     self.store[host, port].Notify(msg)
-    self.write('SENT')
+    self.write('')
```

Comment:
I thought I did that for a reason, but it _does_ seem to work without it, so I'm taking it out.